ampd/src/handlers/multisig.rs

```diff
@@ -120,6 +124,14 @@ where
             result => result.change_context(DeserializeEvent)?,
         };
 
+        if chain.eq(&ChainName::from_str("XRPL").unwrap()) {
```

Comment:
shouldn't have an unwrap here. Also, I am not sure this is the best long term solution, to just hardcode cases here. But doesn't need to be solved in this PR

Reply:
actually, this needs to be configurable right? the chain might not be named "xrpl", especially in different environments (devnet, etc)